batch/sql/estimated-current.sql

```diff
@@ -1114,11 +1176,21 @@ BEGIN
         n_jobs = n_jobs + expected_n_jobs
       WHERE id = in_batch_id;
 
+      UPDATE job_groups
+      INNER JOIN (
+        SELECT batch_id, job_group_id, CAST(COALESCE(SUM(n_jobs), 0) AS SIGNED) AS staged_n_jobs
+        FROM batches_inst_coll_staging
+        WHERE batch_id = in_batch_id AND update_id = in_update_id
+        GROUP BY batch_id, job_group_id
+      ) AS t ON job_groups.batch_id = t.batch_id AND job_groups.job_group_id = t.job_group_id
+      SET `state` = 'running', time_completed = NULL, n_jobs = n_jobs + t.staged_n_jobs;
```

Comment:
Why the inner join? It has to be `expected_n_jobs`, right? This seems like natural code to be added by the subsequent PR that actually handles parallel creation of job groups.

Reply:
I don't remember why I used an inner join. I don't think it really matters. No, you need to add this query here. The whole point of this PR is to make sure the batches table and job groups table are in sync. If the batch is updated, then the corresponding job group's state must be updated.